compile error in version 1.3.6

trying to compile fontconver but see this error
gcc -Wall -I/usr/local/include/freetype2 -I/usr/include/freetype2 -I/usr/include fontconvert.c -lfreetype -o fontconvert
fontconvert.c: In function 'main':
fontconvert.c:125:32: error: 'TT_INTERPRETER_VERSION_35' undeclared (first use in this function)
  FT_UInt interpreter_version = TT_INTERPRETER_VERSION_35;
                                ^~~~~~~~~~~~~~~~~~~~~~~~~
fontconvert.c:125:32: note: each undeclared identifier is reported only once for each function it appears in
fontconvert.c:126:2: warning: implicit declaration of function 'FT_Property_Set'; did you mean 'FT_Face_Properties'? [-Wimplicit-function-declaration]
  FT_Property_Set( library, "truetype",
  ^~~~~~~~~~~~~~~
  FT_Face_Properties
make: *** [Makefile:8: fontconvert] Error 1

--- a/fontconvert/fontconvert.c
+++ b/fontconvert/fontconvert.c
@@ -23,6 +23,7 @@ See notes at end for glyph nomenclature & other tidbits.
 #include <stdint.h>
 #include <ft2build.h>
 #include FT_GLYPH_H
+#include FT_TRUETYPE_DRIVER_H
 #include "../gfxfont.h" // Adafruit_GFX font structures
 
 #define DPI 141 // Approximate res. of Adafruit 2.8" TFT

